.golangci.yml

```diff
@@ -2,8 +2,9 @@
 run:
 
   timeout: 5m
-  skip-dirs-use-default: true
-
+  skip-dirs-use-default: false
+  skip-dirs:
+    - pkg/helm
```

Comment:
Why are we skipping this? Is it because generated code? If so, I'd appreciate a comment here on the reasons.